### DISCUSSION TITLE
Do logged-in only posts show in RSS feeds?

### DISCUSSION THREAD

**Discussion by markcellus** [bluesky-social/atproto#1991] [Clarifications and "Why?"] [answered]
Hello, a few days ago I received a Bluesky newsletter that it has introduced a new feature that allows users to restrict their public posts to only logged-in users (see copied snippet below). If a user opts to having their public posts seen only by "logged-in users", would those posts also not be shown in the RSS feeds? 

> All posts on Bluesky were already public, but prior to today, our application and many others chose to only allow logged-in users to view them. To ease this transition, we've included an opt-out setting. If you want your profile to continue to be seen only by logged-in users, you can opt out by following these steps:

**Comment by bnewbold** [comment, marked as answer]
The current behavior is that no RSS feed is generated accounts which have opted out from the public web interface. The RSS feed only displays top-level posts for a single account, not reposts, replies, etc.
